todo_app/data/mongodb_items.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+import pymongo
+import os
+from todo_app.models.Item import Item
+
+def get_items():
+    with getDbClient() as client:
+        client = client[os.getenv('MONGO_DB_NAME')]["todo-items"]
+        todoItems = list(client.find({"status": "To Do"}))
+        doingItems = list(client.find({"status": "Doing"}))
+        doneItems = list(client.find({"status": "Done"}))
+        return parseItems(todoItems + doingItems + doneItems)
+
+def change_item_status(itemId, status):
+    with getDbClient() as client:
+        client = client[os.getenv('MONGO_DB_NAME')]["todo-items"]
+        client.update_one({"_id": itemId}, {"$set": {"status": status}})
```

Comment:
This doesn't work currently! The reason I believe is that you're currently matching a string (itemId) against an ObjectId (look in the Data Explorer tab in Azure and you'll see your ids have this type). Both here and for the delete we should adjust those as such:
```suggestion
        client.update_one({"_id": ObjectId(itemId)}, {"$set": {"status": status}})
```
(This relies on `from bson import ObjectId` where bson is a library picked up already as part of pymongo)